model/textparse/openmetricsparse_test.go

```diff
@@ -59,7 +64,7 @@ _metric_starting_with_underscore 1
 testmetric{_label_starting_with_underscore="foo"} 1
 testmetric{label="\"bar\""} 1
 # TYPE foo counter
-foo_total 17.0 1520879607.789 # {xx="yy"} 5`
+foo_total 17.0 1520879607.789 # {id="counter-test"} 5`
```

Comment:
Is this change covered by the commit description?  I couldn't see the relevance.

Reply:
https://github.com/prometheus/prometheus/pull/11984#discussion_r1122515778